lib/sgp30/crc.ex

```diff
@@ -0,0 +1,24 @@
+defmodule SGP30.CRC do
+  def check(data_crc_tuples) when is_list(data_crc_tuples) do
+    data_crc_tuples
+    |> Enum.all?(&check/1)
+  end
+
+  def check({data, crc}) do
+    data
+    |> calculate()
+    |> Kernel.==(crc)
```

Comment:
```suggestion
    calculate(data) == crc
```
This is a stylistic nit - I'm not a huge fan of piping to Kernel for simple calls, if you're open to changing it

Reply:
No problem at all.